grails-app/services/de/iteratec/osm/measurement/environment/BrowserService.groovy

```diff
@@ -27,33 +28,32 @@ class BrowserService {
         return Collections.unmodifiableSet(result)
     }
 
-    List<Browser> findAllByNameOrAlias(List<String> browserNameOrAlias) {
+    List<Browser> findAllByNameOrCreate(List<String> browserNames, boolean isHealthCheck = true) {
```

Comment:
The name of this method is misleading as you replaced `Alias` by `Create` but you are still finding them by alias.
Please consider using `findOrCreateByNameOrAlias` instead.